site/config-overrides.md

```diff
@@ -182,13 +182,14 @@ The secret macro `SECRETNAME` field must reference the name of a Kubernetes secr
 **Check each item below to ensure custom situational configuration takes effect:**
 
 * Reference the name of the current bean and each parent bean in any hierarchy you override.
+  * Note that the `combine-mode` verbs (`add` and `replace`) should be omitted for beans that are already defined in your original domain home configuration.
   * See [Override template samples](#override-template-samples) for examples.
 * Use situational config `replace` and `add` verbs as follows:
-  * If you are adding a new bean that doesn't already exist in your original domain home `config.xml`, specify `add` on the MBean itself and on each attribute within the bean. 
+  * If you are adding a new bean that doesn't already exist in your original domain home configuration, specify `add` on the MBean itself and on each attribute within the bean. 
```

Comment:
If you are adding a new bean that doesn't already exist in your original domain home configuration, specify -> 
If you are adding a new bean that doesn't already exist in your original domain home configuration, then specify
(globally, **If** something..., **then** something...)